gcp/api/server.py

```diff
@@ -165,10 +169,32 @@ def build_determine_version_result(
   idx_futures = ndb.get_multi_async([b[0] for b in bucket_match_items])
   output = []
 
+  # Apply bitwise NOT to the user bitmap
+  inverted_empty_bucket_bitmap = (1 << _BUCKET_SIZE) - 1 - empty_bucket_bitmap
+  empty_bucket_count = inverted_empty_bucket_bitmap.bit_count()
+
   for f in idx_futures:
     idx: osv.RepoIndex = f.result()
+
+    # Byte order little is how the bitmap is stored in the indexer originally
+    bitmap = int.from_bytes(idx.empty_bucket_bitmap, byteorder='little')
+
+    # We are looking to find cases where the bitmap generated by the user query
+    # gives a 0, but the bitmap of the repo is a 1. We do not want to count
+    # cases where bitmap of the repo is a 0, and the user query returns is 1,
```

Comment:
nit: grammar is a little confusing from the second sentence onwards. 

Something like:

```
We do not want to count cases where the repo bitmap contains 0 but the user bitmap
contains 1, since these are already accounted for by not having these in the query
results in the first place.
```
 
seems clearer? (assuming I'm understanding this correctly).